tests/grammar/test_dict_grammar.py

```diff
@@ -1,48 +0,0 @@
-import pytest
-from pytest import (
-    raises,
-)
-
-from lark.exceptions import (
```

Comment:
@fubuloubu  why remove these tests ?

Reply:
They no longer compile without more boilerplate because of how I changed the grammar. I could add the boilerplate to get them to compile, but I think some of them will be illegal syntax now. I stripped a lot out of the python setup